fix(ci) - remove unsupported node versions from workflows and speed things up (#775)
privacy-scaling-explorations/maci: README.md

### Requirements

> You should have Node.js (version `=18`) installed. Use `nvm` to install it and manage versions. This code has been tested with all version before Node.js v18 (14 and 16), however we do recommend using Node.js v18.

Reviewer: ```suggestion
You should have Node.js v18 installed. Use `nvm` to install it and manage versions. This code has been tested with earlier Node versions (14 and 16), however we do recommend using Node.js v18.
```
Nitpick but simplifies a bit
